plugins/content_distribution/lib/kContentDistributionManager.php

```diff
@@ -1121,4 +1092,47 @@ public static function addAssetIdsToEntryDistribution($entryDistribution, $distr
 		$entryDistribution->save();
 		return $ret_val;
 	}
+
+	private static function checkIfSunriseIsLater($entryDistribution, $distributionProfile)
```

Comment:
Sunrise not Z

Sent from my iPhone

On 28 ????? 2016, at 10:28, Moshe Maor <notifications@github.com<mailto:notifications@github.com>> wrote:


In plugins/content_distribution/lib/kContentDistributionManager.php<https://github.com/kaltura/server/pull/4545#discussion_r72575514>:

> @@ -1121,4 +1092,47 @@ public static function addAssetIdsToEntryDistribution($entryDistribution, $distr
>               $entryDistribution->save();
>               return $ret_val;
>       }
> +
> +     private static function checkIfSunriseIsLater($entryDistribution, $distributionProfile)


rename shouldWaitForSunrize()

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/kaltura/server/pull/4545/files/a768c54b661575e50b6534a2643abedc9b0d8de9#r72575514>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABtRXZbVtKKsEobS1GH7NTIRHkHfsWukks5qaFoFgaJpZM4JWPK9>.